src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +205,46 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A textarea is intended for entering multi-line texts.
+	 *   composition: >
+	 *      Textarea fields will render an textarea HTML tag.
```

Comment:
Should we mention the byline and the line showing how many characters are used ( if limit is set) ?